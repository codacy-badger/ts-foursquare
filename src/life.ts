interface ILifeResult {
  status: string
}

export const life = (): ILifeResult => ({ status: 'ok' })
