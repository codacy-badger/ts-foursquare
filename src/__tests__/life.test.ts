import { life } from '../life'

test('life', () => {
  expect(life()).toEqual({ status: 'ok' })
  expect(life()).toMatchSnapshot()
})
