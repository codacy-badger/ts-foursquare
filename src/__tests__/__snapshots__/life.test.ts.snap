// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`life 1`] = `
Object {
  "status": "ok",
}
`;
